polaris-react/src/utilities/css.ts

```diff
@@ -51,3 +51,31 @@ export function getResponsiveProps(
     ]),
   );
 }
+
+export type ResponsiveValue =
+  | undefined
+  | string
+  | {
+      [Breakpoint in BreakpointsAlias]?: string;
+    };
+
+export function getResponsiveValue(
+  componentName: string,
+  componentProp: string,
+  responsiveProp?: ResponsiveValue,
+) {
+  if (!responsiveProp) return {};
```

Comment:
Can we just make this property non-optional?

Reply:
We'd be able to avoid a function call if this wasn't present and wouldn't need the early return. Minor thing, this is ok as is